app/frontend/stylesheets/bootstrap/_card.scss

```diff
@@ -4,6 +4,16 @@
   width: 100%;
   position: relative;
 
+  .card-title {
```

Comment:
I'm assuming this is overrides for Bootstrap and that's why it isn't BEM? :) If it can be BEM that would be good

Reply:
It is overwriting bootstrap styles

Reply:
Ah I see, cool yes I think it is fine if we're overriding the overriding bootstrap card styling globally.  I think I'm getting confused because you've got elements included lower down in this file.  They don't look like bootstrap overrides. Maybe not for this story, but they should probably go in a BEM block somewhere else then.